packages/grid/_modules_/grid/components/column-header-item.tsx

```diff
@@ -11,7 +11,7 @@ import { OptionsContext } from './options-context';
 interface ColumnHeaderItemProps {
   column: ColDef;
   colIndex: number;
-  onResizeColumn?: (c: any) => void;
+  onResizeColumn: any;
```

Comment:
why do we remove type here?

Reply:
Otherwise, I would need to duplicate the information in each forwarding layer. There a 4 if I recall correctly. From my perspective, we can discard the type on this one. What solution would you recommend?

Reply:
putting the type as it allows type safety check and avoid some potential issue. Ideally without any at all even on the arguments.
You could do something like `type ResizeColumnHandler = (c: ??)=> void`
And then in case the type changes you just need to update your `ResizeColumnHandler`

Reply:
Ok, I think I have solved the problem.